### PR TITLE
pkg, supernode: cleanup code based on static check warning

### PR DESCRIPTION
**Pull request by hwdef**
Signed-off-by: hwdef <hwdef97@gmail.com>

### Ⅰ. Describe what this PR did

cleanup code based on static check warning

### Ⅱ. Does this pull request fix one issue?

NONE

### CODE REVIEW

File: supernode/daemon/mgr/cdn/manager.go [hunk 1]
```diff
@@ -158,7 +158,7 @@ func (cm *Manager) TriggerCDN(ctx context.Context, task *types.TaskInfo) (*types
 
 	realMD5 := reader.Md5()
 	success, err := cm.handleCDNResult(ctx, task, realMD5, httpFileLength, downloadMetadata.realHTTPFileLength, downloadMetadata.realFileLength)
-	if err != nil || success == false {
+	if err != nil || success {
```
Comment (inoc603) [inline supernode/daemon/mgr/cdn/manager.go]: I think this should be `!success`

Comment (hwdef) [reply supernode/daemon/mgr/cdn/manager.go]: Thank you,
This is my mistake.

**Comment by allencloud** [comment]
LGTM

**Comment by yeya24** [comment]
@hwdef  Where do you find the static check warning? Can you add this feature into our ci?

**Comment by hwdef** [comment]
@yeya24 
I use staticcheck.
And I found @SataQiu already added this feature into ci. #809

**Pull request merged**